less/pages/index.less

```diff
@@ -0,0 +1,13 @@
+
+.page {
+
+  &__title {
+    .textGradient();
+    .cursiveFont(700);
+    width: 100%;
+    margin: 4rem auto;
+    text-align: center;
+    z-index: 1;
+  }
+
+}
```

Comment:
Awesome job on this. I love the animations and the whole layout. Great use of mixins and variables